fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalTreeAnchor.java

```diff
@@ -0,0 +1,21 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+package com.starrocks.sql.optimizer.operator.logical;
+
+import com.starrocks.sql.optimizer.ExpressionContext;
+import com.starrocks.sql.optimizer.base.ColumnRefSet;
+import com.starrocks.sql.optimizer.operator.OperatorType;
+
+/*
+ * For rewrite task to anchor all tree
+ */
+public class LogicalTreeAnchor extends LogicalOperator {
+    public LogicalTreeAnchor() {
+        super(OperatorType.LOGICAL);
+    }
+
+    @Override
+    public ColumnRefSet getOutputColumns(ExpressionContext expressionContext) {
+        return null;
```

Comment:
why not empty column ref set?

Reply:
The node just for save root node when rewrite tree, it's a meaningless node